### ISSUE TITLE
Could not see proper notification using 'notify-send' in KDE plasma

### ISSUE THREAD

**Issue by n0noob** [zacanger/alert-node#3]
**Possible bug.**
Tried executing following code snippet:
```
var alert = require('alert-node');
alert('Hello world', 'notify-send');
```

**Expected behavior:**
It should have displayed the _KDE styled notification_

**Observed behavior:**
Default notification box appeared with provided text.

**Environment info (relevant info only)**:
```
  System:
    OS: Linux 5.4 Manjaro Linux
    Shell: 5.0.11 - /bin/bash
  Binaries:
    Node: 13.3.0 - /usr/bin/node
    npm: 6.13.4 - /usr/bin/npm
```

**Comment by zautumnz** [comment]
Thank you for the bug report. I just pushed v2.1.1 which adds an option to use `kdialog`. I'm not a KDE user myself, so I'm not sure how it's supposed to look. Could you try again on the most recent version with `require('alert-node')('Hello world, 'kdialog')` and let me know if that fixes it?

**Comment by n0noob** [comment]
```kdialog``` is giving following output and is working as expected :
![image](https://user-images.githubusercontent.com/7954964/71642993-8e96e300-2cd9-11ea-8915-fed6d45c21a3.png)

where ```notify-send``` is supposed to show this kind of alert:
![image](https://user-images.githubusercontent.com/7954964/71643048-0b29c180-2cda-11ea-9130-deb7aad07042.png)

which is same as when ```kdialog``` is chosen.

If you ever get access to KDE plasma you can try running following command for testing ```notify-send```:
```notify-send "Hello world"```

**Comment by zautumnz** [comment]
Sorry for the delay on triaging this. I should have time this weekend to spin up a VM with KDE to test.

**Comment by zautumnz** [comment]
@n0noob I believe this is fixed in v2.1.3. Please reopen if there's still an issue after upgrading.
